install/connections.csv

```diff
@@ -153,6 +153,8 @@ CSWB,10,White Bluffs,Portable,,2013-07-23T05:20:10Z,2013-11-19T10:00:00Z
 CSWC,10,White Bluffs Solar Site with Comms,,,2013-07-24T05:05:00Z,2013-11-19T10:00:00Z
 CSWS,20,White Bluffs strong-motion,,,2013-07-23T05:20:10Z,2013-11-19T10:00:00Z
 CTND,10,Courtenay,Portable,,2010-09-05T01:35:02Z,2010-09-29T00:00:00Z
+CTPS,20,Centreport,,,2024-09-23T00:00:00Z,9999-01-01T00:00:00Z
+CTPS,21,Centreport,,,2024-09-23T00:00:00Z,9999-01-01T00:00:00Z
```

Comment:
Noting that if `Titan TACCL-PH2` has the same response as `TACCL-N1` , channel number/datalogger pin does not need to be updated.

Reply:
It's been confirmed that the responses are identical